libs/hdf-converters/src/xccdf-results-mapper.ts

```diff
@@ -278,142 +299,149 @@ export class XCCDFResultsMapper extends BaseConverter {
         status: 'loaded',
         controls: [
           {
-            path: 'Benchmark.Group',
+            path: 'Benchmark',
+            pathTransform: getRulesInBenchmark,
             key: 'id',
             tags: {
               cci: {
-                path: 'Rule.ident',
+                path: ['ident', 'reference'],
                 transformer: extractCci
               },
               nist: {
-                path: 'Rule.ident',
+                path: ['ident', 'reference'],
                 transformer: nistTag
```

Comment:
not producing correct output at times.  see that same "crypto policies are means of" rule.  
![image](https://github.com/mitre/heimdall2/assets/2015821/a61fd3f2-24fd-4bf0-a4ab-24e6e6296168)
![image](https://github.com/mitre/heimdall2/assets/2015821/bf452ce6-2646-4f94-8f07-953e94d6a895)
the nist tag list should consist of those 5

Reply:
feel like it's cause we only look for cci's and then the corresponding nist tag.  if there are no cci's provided then we stuff the default nist tags in there (sa-11/ra-5) as a (usually correct but probably not this time) reasonable enough default.

however, if the references list provides actual nist tags as well then we should use those in addition to / instead of the cci ones.

another example of this happening is 'the installed operating system is fips 140-2 certified".  there are two cci's identified from the references list (
![image](https://github.com/mitre/heimdall2/assets/2015821/8a9dd3be-e89e-459c-ab29-27bdce141651)
) and so we get 2 nist tags, but the references section itself includes a hell of a lot more (
![image](https://github.com/mitre/heimdall2/assets/2015821/32665a99-b007-4fdd-9134-51de406c72f1)
)

Reply:
I believe I have now addressed this point.